exercise3.js

```diff
@@ -70,13 +53,30 @@
 
 // Write a function that takes in a DNA string as a parameter and returns an array with the complementary strand. For example, a string of "GCTA" would return an array of ["C", "G", "A", "T"].  
 
-
-
-//your code...
-
-
-
-
+function dnaStrand(strDna) {
+    const arr = [];
+    let char = '';    
+    for(let i = 0; i < strDna.length; i++) {
+        char = strDna.charAt(i);
+        switch(char) {
+            case 'G':
+                arr.push('C', char);
+                break;
+            case 'T':
+                arr.push('A', char);
+                break;
+            case 'g':
+                arr.push('C', char.toUpperCase());
+                break;
+            case 't':
+                arr.push('A', char.toUpperCase());
+                break;
```

Comment:
you need cases for 'C' and 'A' as well. All 4 letters can show up in the same dna strand. But for how you understood the question, you answered it well.